### PR TITLE
Update C/C++ guide

### PR DESCRIPTION
**Pull request by egpbos**

### CODE REVIEW

**Comment by egpbos** [comment]
Just a second please, I'm adding some more :)

File: best_practices/language_guides/ccpp.md [hunk 1]
```diff
@@ -166,6 +169,43 @@ There are some really good CppCon videos about debugging on YouTube.
 * DDD - primitive GUI frontend for GDB.
 * The IDEs mentioned above either have custom built-in debuggers or provide an interface to GDB or LLDB.
 
+
+## Libraries
+Historically, many C and C++ projects have seemed rather hestitant about using external dependencies (see also .
```
Comment (jhidding) [inline best_practices/language_guides/ccpp.md]: Unfinished `(see also.`

**Comment by egpbos** [comment]
Ah, very good suggestions. I also like https://github.com/nlohmann/json, I'll add that as well. I'm currently looking into ZeroMQ, that would go in the same category as TBB, I guess.

**Comment by egpbos** [comment]
Hm, I can't seem to find the hdf5-cpp source, do you have a link?

File: best_practices/language_guides/ccpp.md (hunk 1 above)
Comment (egpbos) [reply best_practices/language_guides/ccpp.md]: Fixed.

**Comment by jhidding** [comment]
> Hm, I can't seem to find the hdf5-cpp source, do you have a link?

https://support.hdfgroup.org/HDF5/doc/cpplus_RM/index.html

**Comment by jhidding** [comment]
Funny library this `Clara`, how do we think about syntax abuse? ;)

**Comment by egpbos** [comment]
Syntax abuse: anything that makes lines shorter is a win imho ;)

**Comment by egpbos** [comment]
Thx for the review @jhidding!

**Comment by egpbos** [comment]
Should have asked you for a review as well, apologies.

**Comment by LourensVeen** [comment]
No worries, I'll try to remember to make a pull request. I'll be writing some C++ again hopefully soon, so I'll have a look at the guide and see what I can use and/or contribute then.

**Pull request merged**